src/App.tsx

```diff
@@ -1,167 +1,37 @@
-import { Loader } from './components/Loader';
-
+import classNames from 'classnames';
 import './App.scss';
-
-export const App = () => (
-  <div data-cy="app">
-    <nav
-      data-cy="nav"
-      className="navbar is-fixed-top has-shadow"
-      role="navigation"
-      aria-label="main navigation"
-    >
-      <div className="container">
-        <div className="navbar-brand">
-          <a className="navbar-item" href="#/">
-            Home
-          </a>
-
-          <a
-            className="navbar-item has-background-grey-lighter"
-            href="#/people"
-          >
-            People
-          </a>
+import { NavLink, Outlet } from 'react-router-dom';
+
+const getLinkClass = ({ isActive }: { isActive: boolean }) =>
+  classNames('navbar-item ', { 'has-background-grey-lighter': isActive });
```

Comment:
The space at the end of 'navbar-item ' is not necessary since 'classNames' will concatenate the classes with spaces. You can remove the trailing space to clean up the code.